nexus/reconfigurator/planning/src/system.rs

```diff
@@ -1271,7 +1271,7 @@ impl Sled {
     ) -> Sled {
         use typed_rng::TypedUuidRng;
         let unique = unique.unwrap_or_else(|| hardware_slot.to_string());
-        let model = format!("model{}", unique);
+        let model = "913-0000019".to_string();
```

Comment:
Is this a cosmo or gimlet model? I see it's used in several places. it would be really nice to have it as a constant so we know to use it where necessary

Reply:
Ah! I see it's for a gimlet. Can it be something like `const DEFAULT_GIMLET_MODEL: &str = "913-0000019"` ?